app/app/Http/Controllers/SocialAuthController.php

```diff
@@ -42,6 +42,7 @@ private function createNewUserWithGoogle($profile) {
 			$newUser->email = $email;
 			$newUser->first_name = $profile->firstName;
 			$newUser->last_name = $profile->lastName;
+			$newUser->password_hash = User::generateSaltedHash('Hhaccess.2017');
```

Comment:
Remove this line.  Don't set any password for a new user authenticated with third party authentication.

I intentionally removed the 123 password for the same reason that I want you to remove this.